packages/portalnetwork/src/networks/nodeLookup.ts

```diff
@@ -82,6 +82,7 @@ export class NodeLookup {
 
         for (const enr of response.enrs) {
           const decodedEnr = ENR.decode(enr)
+          this.network.portal.updateENRCache([decodedEnr])
```

Comment:
Same as content lookup.  Doesn't it make more sense to only add to the cache once we actually make contact with a node?